### ISSUE TITLE
Create ApexUserAdmin app / roles / users

### ISSUE THREAD

**Issue by toddhay-NOAA** [nwfsc-fram/research-catch-db#3]

**Comment by toddhay-NOAA** [comment]
Currently I cannot add Kate or Lynn as they don't appear in the search results.  I'll ping Brendan to find out what is happening here.

**Comment by SaOgaz-NOAA** [comment]
Hi @toddhay-NOAA, is the idea still  to use ApexUserAdmin for the authentication for this. If so, Will pointed out we will need to move this app into boatnet.

**Comment by toddhay-NOAA** [comment]
Yes, we will use ApExUserAdmin for the authentication, but I'm not seeing why this then has to be part of boatnet.  We should be able to authenticate independently.

**Comment by SaOgaz-NOAA** [comment]
Per a discussion offline in a meeting, I will be copying the necessary auth code out of boatnet into the separate project.
